docs/project/2024/george.petre1104/index.md

```diff
@@ -0,0 +1,96 @@
+# RC Car controlled with WIFI
+Remote Controlled Car using WIFI and a phone app. 
+
+:::info 
+
+**Author**: George-Alexandru PETRE  
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/lab-GeorgePetre11
+
+:::
+
+## Description
+
+An RC Car, created using Pico, that can be controlled using a Phone App with the use of WIFI. The car has controls to move forward,backward and sideways (as of this moment, in time there can also be added a sensor that can detect obstacles) 
+
+## Motivation
+
+I choose this project because I wanted to have an RC car that I could control with my phone and the idea of creating it and modifying it to my liking felt like an interesting challenge to take. 
+
+## Architecture 
+
+
+![Arhitecture](Schematic.png)
+
+
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+The main components of the hardware that I used are:
+- Raspberry Pi Pico - The Pico microcontroller is a small, low-cost, high-performance microcontroller board built with the RP2040 chip developed by Raspberry Pi. It features a dual-core ARM Cortex M0+ processor, which provides ample power for handling the logic and control functions of the car.
+- L289N Driver motor - The L289N is a powerful dual full-bridge driver designed to drive inductive loads such as relays, solenoids, DC, and stepping motors. In this project, this driver manages the motors by allowing the Pico microcontroller to run motors with a higher power requirement than the Pico can provide directly. It can handle a significant current load and operates over a wide voltage range, making it ideal for controlling the wheels of the car.
+- Power Supply  - 4 AA batteries
+- 4WD Kit - This represents the chasis of the car. Contains the wheels, motors for the wheels and 2 acrylic plates that make the body of the car.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico WH](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [56 RON](https://ardushop.ro/ro/home/2819-raspberry-pi-pico-wh.html?search_query=pico+wh&results=108) |
+| [Punte H Dubla L298N] | The Driver Motor | [11.53 RON](https://www.sigmanortec.ro/Punte-H-Dubla-L298N-p125423236) |
```

Comment:
Links are missing or brackets are extra.

Reply:
- added the link for L289N
- removed the rest of the square brackets for the components, as they don't have any links